### ISSUE TITLE
[BUG] Wrong rating in Anilist always ends up rating 10

### ISSUE THREAD

**Issue by Zw012d** [Glagan/SyncDex#24]
**Describe the bug**
Wrong rating in Anilist always ends up rating 10

**To Reproduce**
1. Go to Any manga
2. Rate the manga
3. Sync and check rating on anilist
4. See wrong rating
 
**Screenshots**
https://mangadex.org/title/36332/monkey-peak

![Screenshot 2021-01-11 024744](https://user-images.githubusercontent.com/76645315/104135948-0dc4f980-53b9-11eb-9cb9-8bb02bf8eed7.png)

![Screenshot 2021-01-11 024727](https://user-images.githubusercontent.com/76645315/104135911-c8a0c780-53b8-11eb-9925-6ae13279bedc.png)

https://mangadex.org/title/18207

![Screenshot 2021-01-11 024605](https://user-images.githubusercontent.com/76645315/104135970-2fbe7c00-53b9-11eb-93b6-82f23a7bbf3a.png)

![Screenshot 2021-01-11 024504](https://user-images.githubusercontent.com/76645315/104135920-d9e9d400-53b8-11eb-9e9f-ba6e19cab6f0.png)

**Browser**
Chrome (0.1.9)

**Options and logs**
```
{"import":1610255048212,"lastSync":1610313837000,"logs":[],"options":{"acceptLowScore":false,"autoSync":true,"biggerHistory":true,"chapterStatus":false,"chaptersSaved":400,"checkOnStartup":true,"checkOnStartupCooldown":30,"checkOnStartupMainOnly":true,"colors":{"higherChapter":"transparent","highlights":["rgba(82, 190, 90, 0.6)","rgba(107, 177, 95, 0.6)","rgba(55, 168, 61, 0.6)"],"lowerChapter":"rgba(180, 102, 75, 0.5)","nextChapter":"rgba(104, 115, 251, 0.4)","openedChapter":"rgba(28, 135, 141, 0.4)"},"confirmChapter":false,"errorNotifications":true,"favoriteLanguage":"all","groupTitlesInLists":true,"hideHigher":true,"hideLast":false,"hideLower":true,"highlight":true,"iconsSilentAfterSync":true,"linkToServices":true,"mainService":"mal","mdUpdateSyncDex":true,"noReloadStatus":true,"notifications":true,"originalThumbnail":false,"overviewMainOnly":true,"progressInThumbnail":true,"saveOnlyHigher":true,"saveOnlyNext":false,"saveOpenedChapters":true,"separateLanguages":true,"services":["mal","mu","al"],"silentUpdate":false,"subVersion":9,"thumbnail":true,"thumbnailMaxHeight":80,"tokens":{"anilistToken":"set"},"updateMD":true,"updateOnlyInList":false,"useMochi":true,"version":0.1},"saveSync":{"expires":1610326920103,"refresh":"set","service":"Dropbox","token":"set"}}
```

**Comment by Glagan** [comment]
Your rating system is probably not the ``100 Point`` one on Anilist.

![image](https://user-images.githubusercontent.com/8447621/104137115-d8a9ae80-539a-11eb-82aa-fe86fbfe557b.png)

I thought I tried that and Anilist should handle it itself, I'll see what I can do.

**Comment by Zw012d** [comment]
> Your rating system is probably not the 100 Point one on Anilist.

wont if affect mal-sync if i change it to 100 and also each of my services is based on 10 rating 

> I thought I tried that and Anilist should handle it itself, I'll see what I can do.

could you plz give option regarding this in separate tab (i.e. Sevices Settings) to manually override this

**Comment by Glagan** [comment]
I don't know how MAL-Sync works, so I can't answer you, but I don't care about which scoring system you're using on Anilist since their API is pretty good and I can just let them handle that :)

>could you plz give option regarding this in separate tab (i.e. Sevices Settings) to manually override this

I don't know what you mean by that, you can keep using your favorite scoring system on Anilist, SyncDex will receive the same data from Anilist with any types.

The error was on my side, I was using the wrong field, it should be fixed on version ``0.2.0``, you can close this issue if it's working for you.

**Comment by Zw012d** [comment]
>I don't know how MAL-Sync works, so I can't answer you, but I don't care about which scoring system you're using on Anilist since their API is pretty good and I can just let them handle that :)

I didn't know anilist api was that good i guess mal api which is in beta would needs lot of work to catch up :)

> I don't know what you mean by that, you can keep using your favorite scoring system on Anilist, SyncDex will receive the same data from Anilist with any types.

>The error was on my side, I was using the wrong field, it should be fixed on version 0.2.0, you can close this issue if it's working for you.

thanks It is working now for my case so i am closing this issue now
